testsuite/ui/views/common/login.py

```diff
@@ -0,0 +1,31 @@
+"""Login portion which is the same for admin and master"""
+from widgetastic.widget import TextInput, View, Text
+
+from testsuite.ui.widgets.buttons import ThreescaleSubmitButton
+
+
+class LoginWidget(View):
```

Comment:
This is View, not Widget! In addition, I suggest that we name it more expressive .. `LoginForm`